src/frontend/init.js

```diff
@@ -0,0 +1,80 @@
+var Module = {
```

Comment:
Does this file also need a license header?

Reply:
Added it.